README.md

```diff
@@ -6,32 +8,37 @@
 [![Swift 4.0](https://img.shields.io/badge/swift-4.0-red.svg?style=flat)](https://developer.apple.com/swift)
 [![License](https://img.shields.io/badge/license-MIT-lightgrey.svg?style=flat)](https://opensource.org/licenses/MIT)
 
+[About Rocket.Chat](https://github.com/RocketChat/Rocket.Chat/#about-rocketchat)
+
 # Get it from the store
 
-[![Rocket.Chat on Apple AppStore](https://user-images.githubusercontent.com/551004/29770691-a2082ff4-8bc6-11e7-89a6-964cd405ea8e.png)](https://itunes.apple.com/us/app/rocket-chat/id1148741252?mt=8)
+[![Rocket.Chat on Apple AppStore](https://user-images.githubusercontent.com/551004/29770691-a2082ff4-8bc6-11e7-89a6-964cd405ea8e.png)](https://geo.itunes.apple.com/us/app/rocket-chat/id1148741252?mt=8)
+
+![Screenshots](https://user-images.githubusercontent.com/193273/35487347-d01d115c-047a-11e8-8de6-942e00eb49b3.png)
 
 # Reporting an Issue
 
 [Github Issues](https://github.com/RocketChat/Rocket.Chat.iOS/issues) are used to track todos, bugs, feature requests, and more.
 
-## The app isn't connecting to your server?
-Make sure your server supports WebSocket. These are the minimum requirements for Apache 2.4 and Nginx 1.3 or greater.
+Also check the commuinty on [open.rocket.chat](https://open.rocket.chat/channel/iosnativeapp), we'd like to help.
+
+## How to report an Issue
 
-# Setting up a development environment
+If you found a bug, please include the current version number (Sidebar > Dropdown > My Account). Please also mention the device. Some features only work with newer versions of Rocket.Chat servers, you can get the current version number (even if you're not admin) via `https://chat.server/api/info`.
 
-In case you're interested in playing around with the code or giving something back, here are some instructions on how to set up your project:
+Please include as much information as you can provide. E.g. try to write down steps to reproduce a bug so we can investigate. If you like to help, see [how to contribute](#Contribute).
 
-1. Install [CocoaPods](https://cocoapods.org)  (note that you will need to install at least Ruby 2.2.3 for this to work)
+Is your bug critical regarding security? [Read here](https://github.com/RocketChat/Rocket.Chat.iOS/blob/develop/SECURITY.md)
 
-  `sudo gem install cocoapods`
+## The app isn't connecting to your server?
+Make sure your server supports WebSocket as well as SSL. Apache supports WebSockets from 2.4 on, Nginx with 1.3 or greater.
 
-2. Install all the Pods  (note that this will `git clone` the [CocoaPods github spec repos](https://github.com/CocoaPods/Specs) which may take up to an hour on slow link/machines)
+# Contribute
 
-  `pod install`
+You'd like to help and having a Mac? Or reported some issues and check if they are really solved with a pull request in the beta? Great! We have written down a [Contribution guide](https://github.com/RocketChat/Rocket.Chat.iOS/blob/develop/CONTRIBUTING.md) so you can start easily.
```

Comment:
I think this would be better worded like:

> Are you an iOS developer and would like to help? Found a bug that you would like to report or a missing feature that you would like to request? Great! We have written down a [Contribution guide](https://github.com/RocketChat/Rocket.Chat.iOS/blob/develop/CONTRIBUTING.md) so you can start easily.

Reply:
👍

Reply:
Could you please also update this text as @cardoso suggested?